src/sfizz/Voice.cpp

```diff
@@ -65,16 +65,20 @@ void sfz::Voice::startVoice(Region* region, int delay, int number, float value,
             wave = resources.wavePool.getWaveSaw();
             break;
         }
+        const int quality = region->oscillatorQuality ? *region->oscillatorQuality : Default::oscillatorQuality;
```

Comment:
I think you can use the `optional` function `region->oscillatorQuality.value_or(Default::oscillatorQuality)` here. Does the same thing, and slightly less verbose.